app/views/templates/news_article.html.haml

```diff
@@ -3,26 +3,30 @@
 
 - breadcrumb :public_news_article, node
 
-%div.list-horizontal
-  = render partial: 'templates/related_sections', locals: {node: node}
-
 %h1
   =node.name
 
 %div.abstract
   %p
     =node.short_summary
     - if node.summary
-      != markdown_content node.summary
+      .summary
+        != markdown_content node.summary
 
 %ul.announcement-info
   - if node.release_date
-    %li Published date: #{l(node.release_date, format: :news)}
+    %li
+      %span
+        Date published
+      #{l(node.release_date, format: :news)}
   %li
-    Published by:
-    %a= link_to(node.section.name, public_node_path(node.section.home_node))
-
+    %span
+      Published by
+    %a= link_to(node.section.name, section_news_articles_path(section: @section.home_node.slug))
 
 = render partial: 'templates/content_body', object: node.content_body
 
 =@hierarchy
+
+-#%div.list-horizontal
+  = render partial: 'templates/related_sections', locals: {node: node}
```

Comment:
I agree with you re related news articles but the `related_sections` template here is not rendering related news articles. It's rendering the additional sections (departments, ministers etc) that contributed to the news article.
![image](https://cloud.githubusercontent.com/assets/4583474/18259566/a7858ca8-7428-11e6-87eb-b575d9ac31f2.png)
I'm wondering where this should go on the page. Note: the keyword 'Tags' here was made up by me but really it's just the list of related sections.